City-of-Helsinki/drupal-helfi-platform-config
# UHF-2678 Form tool configurations

How to test:
If you **don't** have a site for testing this feature, create one:
  * `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  * `cd hel-platform`
  * `make new`

If you **do** have a site, get it up and running:
* `make new` or `make fresh` (depending whether you have dump.sql or not)

When the site is up and running:
* Retrieve Form tool configurations `composer require drupal/helfi_platform_config:dev-UHF-2678_form_tool_configurations --with-all-dependencies`
* Install HELfi Form tool config module.
  * `make shell`
  * `drush en -y helfi_media_formtool_config`
  
Log in to the site via the login link
* Go to create basic page.
* Give the page a title and add a new **Form tool embed** paragraph to Upper content region
   * In Form tool embed click on Add media
   * To Form embed code (may also be called Form embed URL) add this code:
   `<script src="//nginx-lomaketyokalu-dev.agw.arodevtest.hel.fi/fi/webform/testilomake/share.js"></script>`
   * and click on Add, then Save the media by clicking on Save and finally Insert the media by clicking Insert selected
* Save the node and check that the form gets rendered
